ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -0,0 +1,565 @@
+{-# LANGUAGE DeriveGeneric         #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE LambdaCase            #-}
+{-# LANGUAGE NamedFieldPuns        #-}
+{-# LANGUAGE OverloadedStrings     #-}
+
+-- | Data types and generators that convert a 'BlockTree' to a 'PointSchedule'.
+--
+-- Point schedules can have arbitrary configurations that model different behaviors
```

Comment:
This comment seems too abstract, in the sense that it might describe many things :) (eg "have arbitrary configurations that model different behaviours". Could we be more specific here? For instance, can we add a table similar to this one:

![image](https://github.com/input-output-hk/ouroboros-consensus/assets/175315/b9b9392c-793c-4286-9aad-8763eda605d8)


Also examples would be really helpful :+1:

Reply:
This module is the focus of the next deliverable. I propose that we collect the comments to address them on the next PR.